components/AppFooter.vue

```diff
@@ -21,7 +21,12 @@
     <div
       class="flex justify-between lg:justify-start items-stretch flex-wrap gap-6 gap-y-1 leading-5"
     >
-      <span class="text-(--ui-text-muted)">© {{ new Date().getFullYear() }} 3ook.com </span>
+      <a
+        class="text-(--ui-text-muted)"
+        href="https://github.com/likecoin/3ook-com?tab=GPL-3.0-1-ov-file#readme"
+        target="_blank"
+        rel="noopener noreferrer"
+      >{{ $t("footer_license") }}</a>
```

Comment:
Please use `<ULink />` and add underline
```suggestion
      <ULink
        class="border-b leading-5"
        to="https://github.com/likecoin/3ook-com?tab=GPL-3.0-1-ov-file#readme"
        target="_blank"
        v-text="$t('footer_license')"
      />
```

Reply:
`<ULink/>` use `<NuxtLink/>` under the hood, so a external link will be added `rel="noopener noreferrer"` automatically
https://nuxt.com/docs/api/components/nuxt-link#external-routing